.circleci/config.yml

```diff
@@ -1,52 +1,53 @@
-# Python CircleCI 2.0 configuration file
-#
-# Check https://circleci.com/docs/2.0/language-python/ for more details
-#
 version: 2
+general:
+  branches:
+    ignore:
+      - gh-pages
 jobs:
   test:
     docker:
-      - image: circleci/python:3.6.1
+      - image: python:3.6-alpine
     working_directory: ~/serum
     steps:
       - checkout
       - run:
-          name: install dependencies
+          name: Prepare
           command: |
-            python3 -m venv venv
-            . venv/bin/activate
-            pip install codecov
+            pip install pipenv
+            pipenv sync -d
       - run:
-          name: run tests and report metrics
+          name: Test
           command: |
-            . venv/bin/activate
-            coverage run --source serum --branch  -m unittest
-            coverage report --fail-under 100
-            codecov
-
-
+            pip install pipenv
+            pipenv sync -d
```

Comment:
I think the pipenv is preserved from the last step, so you don't have run the first two lines again

Reply:
good catch, I thought that this is in a job and not a step :laughing: